com.ibm.streamsx.metrics/com.ibm.streamsx.metrics.service/MetricsIngestService.spl

```diff
@@ -32,38 +32,39 @@
 namespace com.ibm.streamsx.metrics.service;
 
 use com.ibm.streamsx.metrics::* ;
-use com.ibm.streamsx.json::TupleToJSON ;
+use com.ibm.streamsx.json::* ;
 use com.ibm.streamsx.topology.topic::Publish ;
 
 public composite MetricsIngestService
 {			
 	param
-		expression<rstring> $applicationConfigurationName ;
+		expression<rstring> $applicationConfigurationName : getSubmissionTimeValue("applicationConfigurationName", "metricsConfig");
```

Comment:
My mistake, reading the code again and it looks like it does it every *scanPeriod* interval. The default *scanPeriod* is 5 seconds.